pkg/antctl/raw/multicluster/common/cleanup.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Common codes for both leader and member cluster to clean up the Antrea
+// Multi-cluster resources in the given Namespace.
+
+package common
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+type CleanOptions struct {
+	Namespace  string
+	ClusterSet string
+}
+
+func (o *CleanOptions) validate() error {
+	if o.ClusterSet == "" {
+		return fmt.Errorf("ClusterSet is required")
+	}
+
+	return nil
+}
+
+func Cleanup(cmd *cobra.Command, cleanOpts *CleanOptions) error {
+	var err error
+	if err = cleanOpts.validate(); err != nil {
+		return err
+	}
+	k8sClient, err := NewClient(cmd)
+	if err != nil {
+		return err
+	}
+	err = deleteClusterClaims(cmd, k8sClient, cleanOpts.Namespace)
+	if err != nil {
+		fmt.Fprintf(cmd.OutOrStdout(), "Failed to delete ClusterClaim: %v\n", err)
```

Comment:
First, we should remove the logs. Otherwise, we got duplicated log messages.

I meant just to remove "err =", so we have:
```
deleteClusterClaims(cmd, k8sClient, cleanOpts.Namespace)
deleteClusterSet(cmd, k8sClient, cleanOpts.Namespace, cleanOpts.ClusterSet)
...
```

Will this really get an error of golang checking? What is the error?

Reply:
Removed the `err=` now.

Just removing the `err=` will not cause an error. I use the code below which caused an error.
```go
err= deleteClusterClaims()
err= deleteClusterSet()
```